app/config.go

```diff
@@ -88,12 +88,12 @@ type ServerToml struct {
 // Group holds the Roster and the server-description.
 type Group struct {
 	Roster      *onet.Roster
-	description map[*network.ServerIdentity]string
+	Description map[*network.ServerIdentity]string
 }
 
 // GetDescription returns the description of a ServerIdentity.
```

Comment:
Is it really though ? I guess PriFi is not the only protocol where nodes are not all alike